spec/index.bs

```diff
@@ -1229,12 +1242,14 @@ interface GPU {
             1. Let |promise| be [=a new promise=].
             1. Issue the following steps on the [=Device timeline=] of |this|:
                 <div class=device-timeline>
-                    1. If the user agent chooses to return an adapter:
+                    1. If the user agent chooses to return an adapter, it should:
 
-                        1. The user agent chooses an [=adapter=] |adapter| according to the rules in
+                        1. Choose an [=adapter=] |adapter| according to the rules in
                             [[#adapter-selection]] and the criteria in |options|.
 
-                        1. |promise| [=resolves=] with a new {{GPUAdapter}} encapsulating |adapter|.
+                        1. Set `navigator.gpu.`{{GPU/[[adapters_active]]}} to `true`.
```

Comment:
ok, so if the user wants to ignore this logic entirely, they'd just
  - hold onto `GPUAdapter` after a device is lost
  - call `requestAdapter` without bothering to check the result
  - use the old `GPUAdapter` like nothing happened

Isn't the intent here to prevent this behavior? It looks to me that some users may do just this.

Reply:
Yes, they could do this, but it would be harder to do that accidentally.

We could be much more strict and just make new GPUAdapter objects, but we'd lose expando properties and break equality comparisons.